assets/app/view/game/game_log.rb

```diff
@@ -1,19 +1,21 @@
 # frozen_string_literal: true
 
 require 'view/game/actionable'
-require 'view/log'
+require 'lib/settings'
```

Comment:
why did you get rid of the Log import?

Reply:
The rewind feature caused the chat log and the game log to diverge enough that I split them up. Log no longer used here.